type_description_interfaces/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+cmake_minimum_required(VERSION 3.5)
+
+project(type_description_interfaces)
+
+# Default to C++14
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 14)
+endif()
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-Wall -Wextra -Wpedantic)
+endif()
+
+find_package(ament_cmake REQUIRED)
+# Depend on "core" generators instead of "default" generators
+# because other generators depend on this.
+find_package(rosidl_core_generators REQUIRED)
+
+rosidl_generate_interfaces(${PROJECT_NAME}
+  "msg/Field.msg"
+  "msg/FieldType.msg"
+  "msg/IndividualTypeDescription.msg"
+  "msg/TypeDescription.msg"
+  ADD_LINTER_TESTS
+)
+
```

Comment:
```suggestion

if(BUILD_TESTING)
  find_package(ament_lint_auto REQUIRED)
  ament_lint_auto_find_test_dependencies()
endif()
```

Reply:
I'll have to double check these changes, because I think these packages might be kind of special since they're low level message packages. I'll double check.